src/frontend/src/lib/constants/utils.constants.ts

```diff
@@ -0,0 +1,3 @@
+import { BigNumber } from '@ethersproject/bignumber';
+
+export const ZERO = BigNumber.from(0n);
```

Comment:
Maybe move it to `app.constants`? `utils.constants` feel like a constant that is only used in the "utils" modules. A bit an opiniated feedback, hope it makes sense?

Reply:
agreed, changed